sanic/response.py

```diff
@@ -319,9 +319,26 @@ def html(
     )
 
 
+async def validate_file(
+    request_headers: Header, last_modified: Union[datetime, float, int]
+):
+    if_modified_since = request_headers.get("If-Modified-Since")
+    if not if_modified_since:
+        return
+    if_modified_since = parsedate_to_datetime(if_modified_since)
+    if not isinstance(last_modified, datetime):
+        last_modified = datetime.fromtimestamp(
+            float(last_modified), tz=timezone.utc
+        ).replace(microsecond=0)
+    if last_modified <= if_modified_since:
+        return HTTPResponse(status=304)
+
+
 async def file(
     location: Union[str, PurePath],
     status: int = 200,
+    request_headers: Optional[Header] = None,
+    validate_when_requested: bool = True,
```

Comment:
Should this default to true? It's a breaking change, no?

Reply:
Yes, kind of? Maybe make it to be False for now? And just realized that I forgot to add a doc for the added parameters 😂. Will add them soon.

Reply:
And it is how flask and our static files serving work. So still a reasonable behavior?

Reply:
Changed to `False`. The default behavior should remain same as previous.